### PR TITLE
Further clarify reporting of security issues

### PR DESCRIPTION
**Pull request by gus-asf**

### CODE REVIEW

File: content/pages/security.md [hunk 1]
```diff
@@ -5,27 +5,37 @@ template: security
...
-If you **don't** see a CVE there, you should take the following steps:
+1. Check further down this page to see if the CVE is listed as exploitable in Solr.
+2. Check the [officially published non-exploitable vulnerabilities](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) list to see if the CVE is listed as not exploitable in Solr
+3. Search through the [Solr users mailing list archive](https://lists.apache.org/list.html?users@solr.apache.org)  to see if anyone else has brought up this dependency CVE.
+4. If no one has, then please do [subscribe to the users mailing list](https://solr.apache.org/community.html#mailing-lists-chat) and then send an email asking about the CVE.
 
-1. Search through the [Solr users mailing list](https://lists.apache.org/list.html?users@solr.apache.org) to see if anyone else has brought up this dependency CVE.
-1. If no one has, then please do [subscribe to the users mailing list](https://solr.apache.org/community.html#mailing-lists-chat) and then send an email asking about the CVE.
+### Dos and Don'ts
```
Comment (HoustonPutman) [inline content/pages/security.md:18]:
```suggestion
#### Dos and Don'ts
```

Sub section maybe?

**Review by risdenk — APPROVED** [review]

File: content/pages/security.md [hunk 2]
```diff
@@ -5,27 +5,37 @@ template: security
...
+* Please DO discuss the possible need for library upgrades on the user list. 
+* Please DO search Jira for the CVE number to see if we are addressing it already.
+* Please DO create Jira issues and associated pull requests to propose and discuss upgrades of *a single specific* dependency.
+* Please DO NOT attach a scan report, or paste output of a scan into Jira (just link the CVE instead)
+* Please DO NOT email the security email below with a scan report it will be ignored.
 
-### Exploits found in Solr
+### Use of Jira
```
Comment (HoustonPutman) [inline content/pages/security.md:25]:
```suggestion
#### Use of Jira
```

Again, maybe a subsection of the dependency CVEs?

File: content/pages/security.md [hunk 3]
```diff
@@ -5,27 +5,37 @@ template: security
...
 
-The Solr PMC greatly appreciates the reporting of security vulnerabilities found in Solr itself.
+### New Exploits <span style="color:blue">You</span> Discover in Solr
 
-Then please disclose responsibly by following [these ASF guidelines](https://www.apache.org/security/) for reporting.
-You may file your request by email to <mailto:security@solr.apache.org>.
+The Solr PMC greatly appreciates the reporting of new security vulnerabilities found in Solr itself or demonstrations of exploit vulnerability via dependencies. **It is important not to publish a previously unknown exploit**, or exploit demonstration code on public mailing lists. Please disclose new exploits responsibly by following these [ASF guidelines](https://www.apache.org/security/) for reporting. The contact email for reporting newly discovered exploits in Solr is <mailto:security@solr.apache.org>.
```
Comment (HoustonPutman) [inline content/pages/security.md:30]:
```suggestion
The Solr PMC greatly appreciates reports of new security vulnerabilities found in Solr itself or demonstrations of exploiting vulnerabilities via dependencies.
**It is important not to publish a previously unknown exploit**, or exploit demonstration code on public mailing lists.
Please disclose new exploits responsibly by following these [ASF guidelines](https://www.apache.org/security/) for reporting.
The contact email for reporting newly discovered exploits in Solr is <mailto:security@solr.apache.org>.
```

Switching to one line per sentence and cleaning up the first sentence a bit.

**Review by HoustonPutman — APPROVED** [review]
Small changes, but overall I really like it!

**Pull request merged**